workstation/workstation.go

```diff
@@ -33,12 +35,24 @@ type MountInfo struct {
 }
 
 // NewWorkStation returns workstation depending on the OS
-func NewWorkStation() WorkStation {
-	return newWorkstation()
+func NewWorkStation(disk string) WorkStation {
+	return newWorkstation(disk)
 }
 
 // Stringer method
 func (m *MountInfo) String() string {
-	return fmt.Sprintf("DiskName=%s\n\tdeviceName=%s\n\tdiskNameRaw=%s\n\tdeviceSize=%s",
-		m.diskName, m.deviceName, m.diskNameRaw, m.deviceSize)
+	return fmt.Sprintf("DeviceName=%s\nDiskName=%s\nDiskNameRaw=%s\nDeviceSize=%s",
+		m.deviceName, m.diskName, m.diskNameRaw, m.deviceSize)
+}
+
+func (w *workstation) printDisks(ws WorkStation) {
+	var err error
+	disks := []*MountInfo{}
```

Comment:
__[gometalinter]__
* [ineffassign] ineffectual assignment to disks

<!-- Comment by [SideCI](https://sideci.com) -->